packages/tic-tac-toe/config/environment.js

```diff
@@ -18,14 +18,20 @@ module.exports = function(environment) {
       }
     },
     TARGET_NETWORK: process.env.TARGET_NETWORK,
-    FIREBASE_PROJECT: process.env.FIREBASE_PROJECT,
-    FIREBASE_API_KEY: process.env.FIREBASE_API_KEY,
     FIREBASE_PREFIX: process.env.FIREBASE_PREFIX,
     WALLET_URL: process.env.WALLET_URL,
     TTT_CONTRACT_ADDRESS: process.env.TTT_CONTRACT_ADDRESS, // Need to inject this similar to start.js
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+    firebase: {
+      apiKey: process.env.FIREBASE_API_KEY,
+      authDomain: `${process.env.FIREBASE_PROJECT}.firebaseapp.com`,
+      databaseURL: `https://${process.env.FIREBASE_PROJECT}.firebaseio.com`,
+      projectId: process.env.FIREBASE_PROJECT,
+      storageBucket: `https://${process.env.FIREBASE_PROJECT}.appspot.com`,
+      messagingSenderId: process.env.FIREBASE_MESSAGE_SENDER_ID
```

Comment:
I don't think you need anything other than `databaseURL`

Reply:
In RPS we have all of these except an empty storageBucket.
```ts
const config = {
  apiKey: process.env.FIREBASE_API_KEY,
  authDomain: `${process.env.FIREBASE_PROJECT}.firebaseapp.com`,
  databaseURL: `https://${process.env.FIREBASE_PROJECT}.firebaseio.com`,
  projectId: process.env.FIREBASE_PROJECT,
  storageBucket: '',
  messagingSenderId: '913007764573',
};
```
https://github.com/statechannels/monorepo/blob/f98d3f2b2fb39905d533011a08ff1ef745f92b5b/packages/rps/src/gateways/firebase.ts#L7

Also note that I am using Firestore and not Realtime Database.

Reply:
Why are you using Firestore? `rps` uses the realtime database.

Reply:
Also `rps` doesn't need any of those.

Reply:
Done. Only need `FIREBASE_PROJECT` for:
`projectId` and `databaseURL`